janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLConfigOptions.java

```diff
@@ -144,6 +144,48 @@
             ConfigOption.Type.FIXED,
             64);
 
+    ConfigOption<Integer> LOCAL_CORE_CONNECTIONS_PER_HOST = new ConfigOption<>(
+            CQL_NS,
+            "local-core-connections-per-host",
+            "The number of connections initially created and kept open to each host for local datacenter",
+            ConfigOption.Type.FIXED,
```

Comment:
Any specific reason why this configuration is of type `FIXED`?

Reply:
I'm asking because it does not seem clear to me why this could not be changed later on. Just asking to understand if there is some reason that I'm missing.